compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -263,88 +264,166 @@ struct QuantizeDequantizeWeights final : public luci::CircleNodeMutableVisitor<b
   loco::DataType output_type;
   QuantizationGranularity granularity;
 
-  // Quantize and dequantize input tensors of each node
-  bool visit(luci::CircleNode *node)
+private:
+  // Fake quantize weights (Only u8 quantization is supported)
```

Comment:
I'm not sure but can it be checked in advance if it is for u8 or s16 or something?

Reply:
Yes, it is checked before this pass is called (more specifically, in the circle-quantizer).